src/handler/handlerengine.cpp

```diff
@@ -1327,8 +1327,8 @@ class HandlerEngine::Private : public QObject
 	{
 		config = _config;
 
-		// up to 10 timers per connection
-		RTimer::init(config.connectionsMax * 10);
+		// up to 10 timers per connection, plus an extra 100 for misc
```

Comment:
why 100 ?

Reply:
just some number large enough to handle any non-session timers with potential expansion. currently there's 1 per zmq socket, and 4 per zhttpmanager. I'm guessing handler uses less than 20.